src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -953,6 +1070,35 @@ subroutine read_fv3_netcdf_guess(fv3filenamegin)
           write(6,*)'fv3lam_io_dynmetvars2d_nouv is ',(trim(fv3lam_io_dynmetvars2d_nouv(i)),i=1,ntracerio3d)
       endif      
 
+    if (laeroana_fv3cmaq) then
+      jtracer = 0
+!name_chemvars3d chemguess from anavinfo
+      do i=1,size(name_chemvars3d)
+        vartem=trim(name_chemvars3d(i))
+        if (ifindstrloc(aeronames_cmaq_fv3,trim(vartem)) > 0) then
+          jtracer=jtracer+1
+          fv3lam_io_tracerchemvars3d_nouv(jtracer)=trim(vartem)
+          write(6,*)'the chemvarname ',jtracer,vartem,' is found '
+        else
+          write(6,*)'the chemvarname ',vartem,' is not in aeronames_cmaq_fv3, !!!!!!!!!!'
+        endif
+      enddo
+
+      ntracerchemio3d=jtracer+7
+      fv3lam_io_tracerchemvars3d_nouv(jtracer+1)="pm25at"
+      fv3lam_io_tracerchemvars3d_nouv(jtracer+2)="pm25ac"
+      fv3lam_io_tracerchemvars3d_nouv(jtracer+3)="pm25co"
+      fv3lam_io_tracerchemvars3d_nouv(jtracer+4)="pm2_5"
+      fv3lam_io_tracerchemvars3d_nouv(jtracer+5)="amassi"
+      fv3lam_io_tracerchemvars3d_nouv(jtracer+6)="amassj"
+      fv3lam_io_tracerchemvars3d_nouv(jtracer+7)="amassk"
+
+      if (mype == 0) then
+        write(6,*) ' fv3lam_io_tracerchemvars3d_nouv is',(trim(fv3lam_io_tracerchemvars3d_nouv(i)),i=1,ntracerchemio3d)
+      endif
+
+      endif !laeroana_fv3cmaq
```

Comment:
Please align line 1073 with line 1071 and ensure at least two-space indentation through out the indicated lines for if statements and do loops.